### ISSUE TITLE
Features (jets, spillway, etc..) missing

### ISSUE THREAD

**Issue by fresnoboy** [Crewski/njsPC-HA#41]
Hi.  I just tried the integration in my home asssistant setup.  For some reason, the integration does not expose a set of "features", which are switches for spa jets, deck jets, swim jets, etc...  that show up in screenlogic (I have a screenlogic2 controller as well), and in njspc and visible in the dashpanel view:

![image](https://github.com/Crewski/njsPC-HA/assets/52867314/964cea9c-df5a-429b-838d-81314bee1074)

Is this a setup problem or is this functionality not yet supported?

**Comment by Crewski** [comment]
Functionality should be there, it sounds like maybe there was an error setting things up so it aborted.  Check your logs and see if you see anything.  I'd expect an error (probably a key error).  Let me know what you see.

**Comment by fresnoboy** [comment]
Yes you are right.  There was a problem with the initialization, and the plugin eventually came back with a clone of the first njsPC instance but with the missing devices.  I deleted the original one and the new one seems to be working fine.
